packages/api/amplify_api_dart/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -383,18 +381,11 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
 
   /// Sends registration message on ws channel when connected
   /// else init's connection
```

Comment:
nit: Does it make sense to remove `else init's connection` from this comment as I don't think it does that anymore here?